variants/backend-base/config/template.rb

```diff
@@ -2,8 +2,10 @@
 
 template "variants/backend-base/config/database.yml.tt", "config/database.yml", force: true
 
-template "variants/backend-base/config/secrets.example.yml.tt", "config/secrets.example.yml"
-remove_file "config/secrets.yml"
+copy_file "variants/backend-base/config/secrets.yml", "config/secrets.yml", force: true
+copy_file "variants/backend-base/config/app.yml", "config/app.yml"
+remove_file "config/master.key"
+remove_file "config/credentials.yml.enc"
```

Comment:
We aren't using these but by default Rails creates them **and** puts a `secret_key_base` value in there which will be preferred by Rails if it exists and can be decrypted.